libs/libblocksds/readme.md

```diff
@@ -38,21 +38,27 @@ int main(int argc, char *argv[])
 
 ## 2. Usage in Makefile
 
-At the end of your makefile add the following line:
+At the end of your makefile add the following lines:
```

Comment:
I would mention that this is just an example and that you don't need to error if the file isn't found.